swap/types.go

```diff
@@ -36,6 +38,7 @@ type Cheque struct {
 
 // HandshakeMsg is exchanged on peer handshake
 type HandshakeMsg struct {
+	ChainID         *big.Int
```

Comment:
Do we really need `big.Int` for the `ChainID`?

Reply:
I looked at geth and parity and it seems they restrict the value to the uin64 range. I will change it to uint64 here as well.

Reply:
done